spec/bamboo_cache_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'bamboo_cache'
+
+RSpec.describe BambooCache do
+  before(:each) do
```

Comment:
the Ruby way is something like
`let(:cache) { BambooCache.new }`

You could see that in the other tests